olive/auto_optimizer/regulate_mixins.py

```diff
@@ -104,19 +104,21 @@ def _regulate_fp16(self, pass_config, pass_flows):
         return pass_config, pass_flows
 
     def regulate_data_config(self, pass_config, pass_flows):
-        from olive.workflows.run.config import INPUT_MODEL_DATA_CONFIG
-
-        data_configs = self.data_configs.get(INPUT_MODEL_DATA_CONFIG)
-        if not data_configs:
+        if not self.auto_optimizer_config or self.auto_optimizer_config.disable_auto_optimizer:
```

Comment:
The auto optimizer also allows no data configs in which case it would just return. so you add an extra condition here for it.

Reply:
Done!